tools/dependency-tree-diff/dependency-tree-diff.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env sh
+
+# Exit if any command fails
+set -eu
```

Comment:
👍 

Alternative, shorter way to set those (both work the same, just a matter of preference):
```suggestion
#!/bin/sh -eu
```

Also, we usually prefer using `bash` over `sh` as it's more modern and less surprising behavior than sh for some stuff.

```suggestion
#!/bin/bash -eu
```